Makefile

```diff
@@ -13,22 +13,36 @@ GO_FILES := $(shell find . $(DONT_FIND) -o -type f -name '*.go' -print)
 rollout-operator: $(GO_FILES)
 	GOOS=$(GOOS) GOARCH=$(GOARCH) CGO_ENABLED=0 go build -ldflags '-extldflags "-static"' ./cmd/rollout-operator
 
+rollout-operator-boringcrypto: $(GO_FILES)
+	GOEXPERIMENT=boringcrypto GOOS=$(GOOS) GOARCH=$(GOARCH) CGO_ENABLED=1 go build -ldflags '-extldflags "-static"' ./cmd/rollout-operator
```

Comment:
Assuming that boringcrypto requires CGO.

Reply:
yes, the linked openssl is in C and needs to called via CGO

Reply:
there are more details here https://kupczynski.info/posts/fips-golang/ (is somewhat outdated because boringcrypto now supports arm64 as well as amd64, but the rest is still valid)